### DISCUSSION TITLE
Zoom control

### DISCUSSION THREAD

**Discussion by MADSALONI** [tailwindlabs/tailwindcss#9041] [Help]
Hi..i want to know where i can find zoom control

**Comment by wongjn** [comment]
What do you mean by "zoom control"?

**Reply by MADSALONI** [reply]
something like this
![zoom control](https://user-images.githubusercontent.com/110161986/183261066-fcb71aad-c207-41dc-aff3-8d0ef938582a.png)

**Reply by wongjn** [reply]
Tailwind is not a UI library and thus does not have these kind of UI components, but you can use Tailwind to build them.

**Reply by MADSALONI** [reply]
so can you please suggest where i an get this component

**Reply by wongjn** [reply]
You can try to see if there is something in [Tailwind UI](https://tailwindui.com/components), otherwise this isn't really a Tailwind question anymore.
